gradle/libs.versions.toml

```diff
@@ -1,21 +1,25 @@
 [versions]
-managed-liquibase = "4.17.2"
+managed-liquibase = "4.16.0"
```

Comment:
4.16.0 is the version we released with 5.5.0

Versions 4.17+ don't work as they changed how changesets are looked up and loaded

https://docs.liquibase.com/concepts/changelogs/how-liquibase-finds-files.html